### ISSUE TITLE
Create Datasets with Tissue Samples

### ISSUE THREAD

**Issue by chris263** [solgenomics/sgn#3361]
Expected Behavior 
--------------------------------------------------------------------------

Currently, it is possible to have datasets only with plot or plant level.

![Screen Shot 2021-02-25 at 10 38 16 AM](https://user-images.githubusercontent.com/51970212/109177551-ed2ce500-7755-11eb-869a-73b3e1b8b9c2.png)

For Bugs:
---------

**Comment by lukasmueller** [comment]
Should we also add a dataset categories of 'Analysis' (for analysis trials?)

**Comment by chris263** [comment]
Yes, it makes sense to have it available for analysis. For instance, NIRs can use that to run analysis separated with boiled tissues and raw tissues.

**Comment by lukasmueller** [comment]
Hi Chris, could you take a look at how to implement a new "tissue_samples" dimension in the wizard? Bryan probably knows how to do this if you need info. Maybe we should have a short hackathon about it.

**Comment by chris263** [comment]
Duplicated with #5292
